.github/workflows/tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         os: [ubuntu-latest, macos-latest]
         # Test other versions of Python in special cases to avoid exploding the
         # matrix size; make sure to test all supported versions in some form.
-        python-version: [3.9]
+        python-version: ["3.9"]
         case-name: [defaults]
         numpy-requirement: [">=1.20,<1.21"]
```

Comment:
You probably want to unpin this old Numpy requirement now - it was added during the period when Numpy 1.20 was breaking everything.  You'll probably still want a special case somewhere to test your oldest supported version, though.

Reply:
Agreed. I also want to do this in a separate PR though because I want to be sure numpy 1.22 fixes the MKL plus AVX512 bug first so that we don't get regular failures on certain GitHub Action worker machines again (i.e. those that have CPUs with AVX512 support).